### PR TITLE
Add getMediaLimits to client

### PR DESCRIPTION
**Pull request by Half-Shot**
Part of https://github.com/matrix-org/matrix-doc/pull/1189 and https://github.com/matrix-org/synapse/pull/3184

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -727,6 +727,20 @@ MatrixClient.prototype.getGroups = function() {
     return this.store.getGroups();
 };
 
+/**
+ * Get the room for the given room ID.
```
Comment (turt2live) [inline src/client.js]: *ahem*

File: src/client.js [hunk 2]
```diff
@@ -727,6 +727,20 @@ MatrixClient.prototype.getGroups = function() {
...
+ * This function will return a valid room for any room for which a Room event
+ * has been emitted. Note in particular that other events, eg. RoomState.members
+ * will be emitted for a room before this function will return the given room.
+ * @param {module:client.callback} callback Optional.
+ * @return {module:client.Promise} Resolves with an object containing the limits.
+ */
+MatrixClient.prototype.getMediaLimits = function(callback) {
```
Comment (turt2live) [inline src/client.js]: This should probably just be `getMediaRepoConfiguration` or something, given the opportunity for future expansion.

Comment (t3chguy) [reply src/client.js]: +++1 for magic this super generic

**Comment by Half-Shot** [comment]
Aaaaaanyone? :(

**Pull request merged**